server/index.js

```diff
@@ -2,9 +2,14 @@ const express = require('express');
 
 const app = express();
 const port = 3003;
+const db = require('../data/index.js');
 // const path = require('path');
 // const bodyParser = require('body-parser');
 
 app.use('/', express.static('public'));
 
+app.get('/listings', (req, res) => {
```

Comment:
@VALE46GP This endpoint does not match any of your listed endpoints in your [App Service Plan](https://docs.google.com/document/d/19M5W_dSp8zcd9T9t4eLTKkGgW8KGT3Iwcu6RKzFmRxI/). Is this a different endpoint from the one listed?